GoogleForCreators/web-stories-wp
# Remove Duplicate Icons

## Context


Replaced the original pencil icon with the one in design-system.


## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions



### QA

This is purely cosmetic. 

Go to Dashboard Settings and hover or focus on a publisher logo, the icon should be updated and the button is a little bigger, like this: 
https://user-images.githubusercontent.com/14279973/129640746-e5a28a68-b959-4727-86c9-179f94dea605.png

Please verify that clicking on the button still opens up the context menu as expected and that it looks like this screenshot.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #
